BazelExtensions/extensions.bzl

```diff
@@ -84,6 +84,53 @@ def acknowledged_target(name,
     )
 
 
+
+def _umbrella_header_impl(ctx):
+    headers_list = _get_module_map_headers(ctx.attr.hdrs)
+    output = ctx.outputs.umbrella_header
+    content = """
+#ifdef __OBJC__
+#import <UIKit/UIKit.h>
+#else
+#ifndef FOUNDATION_EXPORT
+#if defined(__cplusplus)
+#define FOUNDATION_EXPORT extern "C"
+#else
+#define FOUNDATION_EXPORT extern
+#endif
+#endif
+#endif
+
+{hdrs}
+
+FOUNDATION_EXPORT double ParentVersionNumber;
+FOUNDATION_EXPORT const unsigned char ParentVersionString[];
```

Comment:
I'm not familiar with these symbols. What's their purpose?

Reply:
I imagine these are vars pulled from a template somewhere, this is boiler plate pulled from the Xcode build, it's possible that some code _may_ use these so I've included it for completeness.